big_tests/tests/push_helper.erl

```diff
@@ -65,6 +65,31 @@ become_unavailable(Client) ->
     escalus:send(Client, escalus_stanza:presence(<<"unavailable">>)),
     {ok, _} = wait_for_user_offline(Client).
 
+become_available(Client, NumberOfUnreadMessages) ->
+    escalus:send(Client, escalus_stanza:presence(<<"available">>)),
+    Preds = [ is_presence | lists:duplicate(NumberOfUnreadMessages, is_message) ],
+    Stanzas = escalus:wait_for_stanzas(Client, NumberOfUnreadMessages + 1),
+    escalus:assert_many(Preds, Stanzas),
+    {ok, true} = wait_for_user_online(Client).
+
+is_online(LUser, LServer, LRes) ->
```

Comment:
Is it the opposite to `is_offline` function from this module?

Reply:
Yes

Reply:
It is an opposite but not in 100%, because the result of `is_online` cannot be achieved by simple negation of `is_offline` result.

Reply:
I think it's enough to have both functions, and call 'not is_online' from 'is_offline'. Or the other way around.

Reply:
@fenek could you give an example?

Reply:
@michalwski I tried putting it into one function like this and it didn't work, that's why I left `is_offline` and andded `is_online`

Reply:
All right, let's leave it as is.